modules/rosa-hcp-create-network.adoc

```diff
@@ -89,4 +364,75 @@ INFO[0140] Resource: VPC, Type: AWS::EC2::VPC, ID: <vpc-id>
 INFO[0140] Stack rosa-network-stack-5555 created \ <2>
 ----
 <1> These two subnet IDs are used to create your cluster when using the `rosa create cluster` command.
-<2> The network stack name is used to delete the resource later.
\ No newline at end of file
+<2> The network stack name is used to delete the resource later.
+
+ifdef::rosa-egress-lockdown[]
+[discrete]
+[id="rosa-hcp-vpc-subnet-tagging-rosa-network_{context}"]
+== Tagging your subnets
+
+Before you can use your VPC to create a {hcp-title} cluster, you must tag your VPC subnets. Automated service preflight checks verify that these resources are tagged correctly. The following table shows how your resources should be tagged:
+
+[cols="3a,8a,8a", options="header"]
+|===
+| Resource
+| Key
+| Value
+
+| Public subnet
+| `kubernetes.io/role/elb`	
+| `1` or no value
+
+| Private subnet 
+| `kubernetes.io/role/internal-elb`	
+| `1` or no value
+
+|===
+
+[NOTE]
+====
+You must tag at least one private subnet and one public subnet, if applicable.
+====
+
+. Tag your resources in your terminal by running the following commands:
+.. For public subnets, run:
```

Comment:
```suggestion
. Tag your resources in your terminal: 
.. For public subnets, run the following command:
```

- to be consistent with modules/rosa-hcp-vpc-manual.adoc, add "run the following command:"